plugins/git-release-manager/src/features/Features.tsx

```diff
@@ -89,6 +90,19 @@ export function Features({
     );
   }
 
+  let CustomFeatures =
+    features?.custom?.factory({
+      latestRelease: gitBatchInfo.value.latestRelease,
+      project,
+      releaseBranch: gitBatchInfo.value.releaseBranch,
+      repository: gitBatchInfo.value.repository,
+    }) ?? null;
+  if (Array.isArray(CustomFeatures)) {
+    CustomFeatures = CustomFeatures.map((CustomFeature, index) => (
+      <Box key={`grm--custom-feature--${index}`}>{CustomFeature}</Box>
```

Comment:
I suggest using `React.Fragment` instead of `Box`, since that won't affect the rendered output.

Another approach might be to do 

```ts
customFeatures.map((child, index) => {
    return React.cloneElement(child, { ...someProps, key: index });
});
```

which I personally think looks odd and wasteful but I think it's actually the most proper solution according to common patterns